calculated_attributes.gemspec

```
@@ -3,25 +3,26 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'calculated_attributes/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = 'calculated_attributes'
-  spec.version       = CalculatedAttributes::VERSION
-  spec.authors       = ['Zach Schneider']
-  spec.email         = ['zach@aha.io']
-  spec.summary       = 'Automatically add calculated attributes to ActiveRecord models.'
-  spec.homepage      = 'https://github.com/aha-app/calculated_attributes'
-  spec.license       = 'MIT'
+  spec.name                  = 'calculated_attributes'
+  spec.version               = CalculatedAttributes::VERSION
+  spec.authors               = ['Zach Schneider']
+  spec.email                 = ['zach@aha.io']
+  spec.summary               = 'Automatically add calculated attributes to ActiveRecord models.'
+  spec.homepage              = 'https://github.com/aha-app/calculated_attributes'
+  spec.license               = 'MIT'
+  spec.required_ruby_version = '>= 3.1.0'
 
   spec.files         = `git ls-files -z`.split("\x0")
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.add_development_dependency 'appraisal', '~> 2.2.0'
-  spec.add_development_dependency 'bundler', '~> 1.15'
-  spec.add_development_dependency 'rake', '>= 12.3.3'
-  spec.add_development_dependency 'rspec', '~> 3.1'
-  spec.add_development_dependency 'rubocop', '~> 0.74.0'
-  spec.add_development_dependency 'sqlite3', '~> 1.3.10'
+  spec.add_development_dependency 'appraisal'
+  spec.add_development_dependency 'bundler'
+  spec.add_development_dependency 'rake'
+  spec.add_development_dependency 'rspec'
+  spec.add_development_dependency 'rubocop'
+  spec.add_development_dependency 'sqlite3'
 
-  spec.add_dependency 'activerecord', '>= 3.2.20'
+  spec.add_dependency 'activerecord', '>= 6.0.0'
```

We should add an upper bound so that rails 8 does not catch us unawares (especially since calculated_attributes raises an exception if someone tries to use the gem with an unsupported rails version).

```suggestion
  spec.add_dependency 'activerecord', '>= 6.0.0', '< 8'
```